deploy-as-code/helm/charts/digit-works/backend/estimate-service/values.yaml

```diff
@@ -80,3 +80,6 @@ env: |
   - name: ESTIMATE_SEARCH_MAX_LIMIT
     value: {{ index .Values "estimate-search-max-limit" | quote }} 
   {{- end }}
+  - name: ESTIMATE_WORKFLOW_MODULE_NAME
+    value: {{ index .Values "estimate-workflow-module-name" | quote }} 
+  {{- end }}
```

Comment:
You have missed writing the starting "if" statement.

Reply:
corrected